src/pages/Login/Login.js

```diff
@@ -1,7 +1,148 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { AlertCircle, ArrowLeft, Eye } from 'react-feather';
+import './Login.scss';
 
 function Login() {
-  return <div>Login.js</div>;
+  const [inputs, setInputs] = useState({
+    userId: '',
+    password: '',
+  });
+
+  const { userId, password } = inputs;
+
+  const navigate = useNavigate();
+
+  const [passwordVisible, setPasswordVisible] = useState(false);
+
+  const [warning, setWarning] = useState({
+    idIncorrect: false,
+    pwIncorrect: false,
+  });
+
+  const { idIncorrect, pwIncorrect } = warning;
+
+  const handleInput = event => {
+    const { name, value } = event.target;
+    event.preventDefault();
+
+    setInputs({
+      ...inputs,
+      [name]: value,
+    });
+  };
+
+  const handleClick = () => {
+    setPasswordVisible(!passwordVisible);
```

Comment:
```suggestion
    setPasswordVisible(prev => !prev);
```
함수형 업데이트 방식으로 구현해주세요!